internal/common/eventutil/eventutil_test.go

```diff
@@ -655,7 +685,6 @@ func TestLimitSequenceByteSize(t *testing.T) {
 
 	_, err = LimitSequenceByteSize(sequence, 1, false)
 	assert.NoError(t, err)
-	assert.Equal(t, []*armadaevents.EventSequence{sequence}, actual)
```

Comment:
Removing this as it is the same as the assert on line 651, and the two variables being compared haven't changed between this line and 651